src/fu-util.c

```diff
@@ -590,7 +593,7 @@ fu_util_get_topology (FuUtilPrivate *priv, gchar **values, GError **error)
 		g_print ("%s\n", _("No hardware detected with firmware update capability"));
 		return TRUE;
 	}
-	fu_util_build_device_tree (root, devs, NULL);
+	fu_util_build_device_tree (root, devs, priv->show_all_devices, NULL);
```

Comment:
Maybe just pass self to `fu_util_build_device_tree`?

Reply:
Sure, switched around.